projects/xerces-c/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#https://github.com/linux-on-ibm-z/docs/wiki/Building-Xerces
+set -e
+cd $SRC/xerces-c
+./reconf
+./configure
+make -j
+
+cd $SRC
+
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE -std=c++11 \
+        -I. -Ixerces-c/src \
+        xerces_fuzz_common.cpp parse_target.cpp -o $OUT/parse_target \
+        xerces-c/src/.libs/libxerces-c.a
+
+if [[ $CFLAGS != *sanitize=memory* ]]; then
```

Comment:
I'm confused is this project able to build with MSAN?
If not, why the special case then?
If yes, why isn't it enabled?

Reply:
The latest commit enables MSAN.